source/client/index.js

```diff
@@ -1,7 +1,12 @@
 const React = require('react')
-const withScroll = require('scroll-behavior').default
+const useScroll = require('react-router-scroll/lib/useScroll')
```

Comment:
Can we change this to `const { useScroll } = requre('react-router-scroll')`?

https://github.com/taion/react-router-scroll#usage